Slack/README.md

```diff
@@ -1,15 +1,18 @@
-# Indexing Slack Using the REST API Connector
```

Comment:
You can probably open a Jira to get rid of this file now that we have a native source.